src/main/java/uk/gov/hmcts/reform/pip/account/management/model/errored/ErroredSubscriber.java

```diff
@@ -1,9 +1,10 @@
-package uk.gov.hmcts.reform.pip.account.management.model;
+package uk.gov.hmcts.reform.pip.account.management.model.errored;
 
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
+import uk.gov.hmcts.reform.pip.account.management.model.Subscriber;
```

Comment:
This can be removed :)

Reply:
its needed and used in here, this is your ErroredSubscriber class which extends the subscriber model, if its deleted the `extends Subscriber` dies